nss-day-cohort-41/react-nutshell-the-cashews
# Bw cashews 2

# Description
Beginning implementation of chat functionality. Users can see messages added by all users listed in descending order and an edit button will display for the user’s own messages (edit functionality has not been implemented yet).
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update
# Testing Instructions
User will need to use json-server and npm start to host both the data and app.
User will need to register a new account, which will be added to the database permanently. 
The user will need to change the component displaying on line 40 <Messages /> to display chat and the user will see all messages in the database as well as be able to add their own.
# Checklist:
